cli/daemon/run/testdata/echo_client/client.ts

```diff
@@ -190,7 +203,11 @@ export namespace echo {
         public async EmptyEcho(params: EmptyData): Promise<EmptyData> {
             // Now make the actual call to the API
             const resp = await this.baseClient.callAPI("POST", `/echo.EmptyEcho`, JSON.stringify(params))
-            return await resp.json() as EmptyData
+            try {
+                return await resp.json() as EmptyData
+            } catch (err) {
+                throw new APIError(500, { code: ErrCode.DataLoss, message: "unable to unmarshal the response", details: err })
```

Comment:
`DataLoss` is not meant for this but rather fatal, permanent data loss on the server side. Change to `Internal`